cypress/e2e/CMS/$SiteSections/index.js

```diff
@@ -0,0 +1,95 @@
+});
+
+And('user types {string} into Landing Page field', (articleName) => {
+    cy.get(`[class*='form-item-field-landing-page-0-target-id']`).find('input#edit-field-landing-page-0-target-id').type(articleName);
+});
+
+And('user selects {string} from landing page dropdown', (item) => {
+    cy.get(`a:contains('${item}')`).first().should('be.visible').click({ force: true });
+});
+
+And('user expands RELATIONS dropdown', () => {
+    cy.get('details#edit-relations').find('span').first().click({ force: true });
+});
+
+And('user selects {string} from RELATIONS dropdown', (selectSiteSection) => {
+    cy.get('#edit-parent').should('be.visible').select(selectSiteSection);
+});
+
+And('user removes site section', () => {
+    cy.get('#edit-field-site-section-current-items-0-remove-button').click();
+});
+
+When('user selects {string} site section', (selectSiteSection) => {
+    cy.get("input[value='Select Site Section']").click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find('input[name="computed_path_value"]').type(selectSiteSection);
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find('input#edit-submit-site-section-browser').click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find(`td:contains('${selectSiteSection}')`).first().parent()
+        .find('td.views-field.views-field-entity-browser-select input').check();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find("input[id='edit-submit'][value='Select Site Section']").click();
+});
+
+Given('user is navigating to the front end site with path {string}', (purl) => {
+    cy.on('uncaught:exception', (err, runnable) => {
+        if (err.message.includes('Failed to load script https://assets.adobedtm.com')) {
+            return false;
+        }
+        return true;
+    })
+    cy.visit(`${frontEndBaseUrl}${purl}`, { retryOnStatusCodeFailure: true });
+});
+
+Then('the current page is {string}', (title) => {
+    cy.get(`li.usa-sidenav__item a:contains("${title}")`).should('have.attr', 'aria-current', 'page');
+});
+
+And('the following nav children are displayed', (dataTable) => {
+    for (let { item } of dataTable.hashes()) {
+        cy.get(`li.usa-sidenav__item a:contains("${item}")`).should('be.visible').and('contain.text', item);
+        cy.wait(2000);
+    }
+});
+
+And('{string} has been selected', (titleText) => {
+    cy.get(`div[id*='edit-field-site-section-current-items-0']`).should('be.visible').and('contain.text', titleText);
+});
+
+And('{int} level up section is {string}', (num, titleText) => {
+    cy.get('ul.usa-sidenav__sublist').parent('li.usa-sidenav__item').find('a').should('contain.text', titleText);
+});
+
+And('user selects a checkbox next to title {string} with url {string} from the list of content', (title, lastPartUrl) => {
+    cy.get(`a:contains("${title}")`).invoke('attr', 'href').then(href => {
```

Comment:
why are you invoking a href attr, but never use it?

Reply:
Removed